### PR TITLE
Further improve multicluster round-robin test

### PR DESCRIPTION
**Pull request by averevki**

### CODE REVIEW

File: testsuite/tests/multicluster/test_simple_strategy.py [hunk 1]
```diff
@@ -26,7 +26,9 @@ def test_simple_strategy(client, hostname, gateway, gateway2):
 
     dns_ip1 = dns.resolver.resolve(hostname.hostname)[0].address
     dns_ip2 = dns.resolver.resolve(hostname.hostname)[0].address
+    dns_ip3 = dns.resolver.resolve(hostname.hostname)[0].address
     assert dns_ip1 != dns_ip2, "Simple routing strategy should return IPs in a round-robin fashion"
+    assert dns_ip1 == dns_ip3, "Simple routing strategy should return IPs in a round-robin fashion"
```
Comment (jsmolar) [inline testsuite/tests/multicluster/test_simple_strategy.py:31]: What about resolving many addresses (10, 50, 100 ??) and verifying if the round-robin is working on that?

**Comment by averevki** [comment]
test

**Comment by averevki** [comment]
test

**Review by azgabur — DISMISSED** [review]

**Review by azgabur — APPROVED** [review]

File: testsuite/tests/multicluster/test_simple_strategy.py [hunk 2]
```diff
@@ -14,20 +14,21 @@ def test_gateway_readiness(gateway, gateway2):
...
 
-    dns_ip1 = dns.resolver.resolve(hostname.hostname)[0].address
-    dns_ip2 = dns.resolver.resolve(hostname.hostname)[0].address
-    assert dns_ip1 != dns_ip2, "Simple routing strategy should return IPs in a round-robin fashion"
+    gw1_ip, gw2_ip = gateway.external_ip().split(":")[0], gateway2.external_ip().split(":")[0]
 
-    gateway_ips = {gateway.external_ip().split(":")[0], gateway2.external_ip().split(":")[0]}
-    dns_ips = {dns_ip1, dns_ip2}
-    assert gateway_ips == dns_ips, f"Expected and actual IPs mismatch, expected {gateway_ips}, got {dns_ips}"
+    for i in range(10):
```
Comment (jsmolar) [inline testsuite/tests/multicluster/test_simple_strategy.py:28]: idea from your previous code. What about checking if the ips are not the same?

**Review by azgabur — APPROVED** [review]

**Review by jsmolar — APPROVED** [review]

**Pull request merged**